### ISSUE TITLE
How to train Kitti_car using cpu mode?

### ISSUE THREAD

**Issue by tomzhang666** [zhaoweicai/mscnn#79]
calvisitor-10-105-153-88:mscnn-7s-576-2x zhangzhengyu$ sh train_mscnn.sh
I0611 12:40:13.772783 3097523136 caffe.cpp:217] Using GPUs 0
F0611 12:40:13.773375 3097523136 common.cpp:66] Cannot use GPU in CPU-only Caffe: check mode.
*** Check failure stack trace: ***
    @        0x10b9f20ce  google::LogMessage::Fail()
    @        0x10b9f13ad  google::LogMessage::SendToLog()
    @        0x10b9f1ab3  google::LogMessage::Flush()
    @        0x10b9f4d03  google::LogMessageFatal::~LogMessageFatal()
    @        0x10b9f23b7  google::LogMessageFatal::~LogMessageFatal()
    @        0x10b6f2651  caffe::Caffe::SetDevice()
    @        0x10b662b07  train()
    @        0x10b665557  main
    @     0x7fffafaf5235  start
    @                0x5  (unknown)
I0611 12:40:13.802539 3097523136 caffe.cpp:217] Using GPUs 0
F0611 12:40:13.803140 3097523136 common.cpp:66] Cannot use GPU in CPU-only Caffe: check mode.
*** Check failure stack trace: ***
    @        0x1094410ce  google::LogMessage::Fail()
    @        0x1094403ad  google::LogMessage::SendToLog()
    @        0x109440ab3  google::LogMessage::Flush()
    @        0x109443d03  google::LogMessageFatal::~LogMessageFatal()
    @        0x1094413b7  google::LogMessageFatal::~LogMessageFatal()
    @        0x109146651  caffe::Caffe::SetDevice()
    @        0x1090b8b07  train()
    @        0x1090bb557  main
    @     0x7fffafaf5235  start

So basically I am using MacBook Pro with OS X 10.12.4, but I don't have a Nividia GPU. How to train the network with CPU mode?

Thanks
